### ISSUE TITLE
Cannot apply patch https://www.drupal.org/files/issues/2023-06-15/1349080-521.patch

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_services#237]
Seeing the following: 

```
  - Applying patches for drupal/core
    https://www.drupal.org/files/issues/2022-06-17/2845144-67.patch (Users can't reference unpublished content even when they have access to it. See https://www.drupal.org/project/drupal/issues/2845144)
    https://www.drupal.org/files/issues/2023-06-15/1349080-521.patch (node_access filters out accessible nodes when node is left joined (1349080))
   Could not apply patch! Skipping. The error was: Cannot apply patch https://www.drupal.org/files/issues/2023-06-15/1349080-521.patch
```

@ekes do we want to update to your patch on https://www.drupal.org/project/drupal/issues/1349080#comment-15334980 ?

**Comment by finnlewis** [comment]
This is coming from our patch here: 

https://github.com/localgovdrupal/localgov_services/blob/2.x/composer.json#L23

**Comment by millnut** [comment]
This also breaks the actions on https://github.com/localgovdrupal/localgov_project due to the patch not applying now the project uses Drupal 10.2

**Comment by millnut** [comment]
Looking at the latest comments on https://www.drupal.org/project/drupal/issues/1349080#comment-15384477 this will be fixed as part of Drupal 10.3, so the patch will only be for Drupal 10.2.
